.github/workflows/main.yml

```diff
@@ -34,7 +35,26 @@ jobs:
           docker build -t $ECR_REPOSITORY:$IMAGE_TAG .
           docker tag $ECR_REPOSITORY:$IMAGE_TAG $ECR_REGISTRY/$ECR_REPOSITORY:$IMAGE_TAG
           docker push $ECR_REGISTRY/$ECR_REPOSITORY:$IMAGE_TAG
+          echo "::set-output name=image::$ECR_REGISTRY/$ECR_REPOSITORY:$IMAGE_TAG"  
 
-      - name: Update ECS service
+      - name: Download task definition
```

Comment:
How did you test this change? Or how are you sure did it work? For configuration changes like this, please attach a Loom in the PR to show that this change works.

Reply:
I replicated the workflow configuration and triggered it to run when I pushed to my working-branch. This allowed me to confirm that the changes work as expected. Attached is a screenshot showing the successful deployment for verification.
![Screenshot 2024-11-13 214337](https://github.com/user-attachments/assets/54d4f2f3-2372-4d0e-aa10-267b9df81c4d)

Reply:
Here's a loom video as well:
https://www.loom.com/share/27ea3050c7b3433e871f832c7af0f498?sid=cf58bd39-0567-4045-baff-3678cb99a76b

You can skip to 1:16 to see that it worked